src/familybot/plugins/steam_family.py

```
@@ -386,16 +395,29 @@ async def check_deals_command(self, ctx: PrefixedContext):
+                    async with aiohttp.ClientSession() as session:
+                        for attempt in range(max_retries):
+                            try:
+                                await self.steam_api_manager.rate_limit_steam_api()
+                                async with session.get(
+                                    wishlist_url,
+                                    timeout=aiohttp.ClientTimeout(total=15),
+                                ) as response:
+                                    response.raise_for_status()
+                                    wishlist_json = await response.json()
+                                    break
+                            except (aiohttp.ClientError, asyncio.TimeoutError) as e:
```

Ruff (0.15.6)

[warning] Replace aliased errors with `TimeoutError`

Replace with builtin `TimeoutError`

(UP041)